lightning_integration_test.py

```diff
@@ -45,17 +52,24 @@
 
 if (actual_proxy_response.returncode != 0):
     print('FAILED: proxy server encountered an error')
+    ec = 1;
 
 if (actual_proxy_response.stdout != expected_proxy_response):
     print('FAILED: proxy server received a non-matching proxy response')
     print('Expected proxy response: \n%s' % expected_proxy_response)
     print('Completed proxy response: \n%s' % actual_proxy_response.stdout.decode('UTF-8'))
+    ec = 1;
 else:
   print('SUCCESS: Received expected reverse proxy response!')
 
 # Terminate the server
 server_process.kill()
 proxy_server_process.kill()
 
-
-print("\nEND Lightning integration tests. All tests passed!")
+# Return 0 if the test succeeded or some other value on failure
```

Comment:
Nice, this will make it much clearer what's actually happening with our integration tests